atlasdb-config/build.gradle

```diff
@@ -47,3 +48,12 @@ dependencies {
     // Needed for Jersey Response-based tests
     testCompile group: 'org.glassfish.jersey.core', name: 'jersey-common'
 }
+
+recommendedProductDependencies {
```

Comment:
What does this do exactly?

Reply:
So if you're shipping a distribution and you include a dependency on this package and you have Gradle Preflight, then you'll fail to build. If you notice on `timelock-server` I've added an ignore on this dependency; if I didn't have that, the build would fail.